catppuccin_tmux.conf

```diff
@@ -4,7 +4,7 @@ source -F "#{d:current_file}/themes/catppuccin_#{@catppuccin_flavor}_tmux.conf"
   set -gF @_ctp_status_bg "#{@thm_mantle}"
```

Comment:
Should default background be `thm_mantle` or `thm_bg`?

Reply:
The main "background pane" is the terminal window (or tmux pane). The status line is a "secondary pane"

https://github.com/catppuccin/catppuccin/blob/main/docs/style-guide.md